### PR TITLE
DBZ-7107 Update documentation

### PR DESCRIPTION
**Pull request by Naros**
Documentation for https://issues.redhat.com/browse/DBZ-7107

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 1]
```diff
@@ -1468,6 +1500,14 @@ a| Optional field that displays the time at which the connector processed the ev
  +
 In the `source` object, `ts_ms` indicates the time that the change was made in the database. By comparing the value for `payload.source.ts_ms` with the value for `payload.ts_ms`, you can determine the lag between the source database update and {prodname}.
 
+|6
+|`ts_us`
+a|Optional field that displays the time at which the connector processed the event, in microseconds. The time is based on the system clock in the JVM running the Kafka Connect task.
+
+|7
+|`ts_ns`
+a|Optional field that displays the time at which the connector processed the event, in nanoseconds. The time is based on the system clock in the JVM running the Kafka Connect task.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:1509]:
```suggestion
a|Optional field that displays the time at which the connector processed the event, in nanoseconds. 
The time is based on the system clock in the JVM that runs the Kafka Connect task.
```

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 2]
```diff
@@ -1642,6 +1698,14 @@ a|Optional field that displays the time at which the connector processed the eve
 +
 In the `source` object, `ts_ms` indicates the time that the change was made in the database. By comparing the value for `payload.source.ts_ms` with the value for `payload.ts_ms`, you can determine the lag between the source database update and {prodname}.
 
+|4
+|`ts_us`
+a|Optional field that displays the time at which the connector processed the event, in microseconds. The time is based on the system clock in the JVM running the Kafka Connect task.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:1703]:
```suggestion
a|Optional field that displays the time at which the connector processed the event, in microseconds. 
The time is based on the system clock in the JVM that runs the Kafka Connect task.
```

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 3]
```diff
@@ -1642,6 +1698,14 @@ a|Optional field that displays the time at which the connector processed the eve
...
+
+|5
+|`ts_ns`
+a|Optional field that displays the time at which the connector processed the event, in nanoseconds. The time is based on the system clock in the JVM running the Kafka Connect task.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:1707]:
```suggestion
a|Optional field that displays the time at which the connector processed the event, in nanoseconds. 
The time is based on the system clock in the JVM that runs the Kafka Connect task.
```

**Comment by Naros** [comment]
> A couple small nitpicks ;)

No worries, I'll split those two sentences across all the documentation onto two lines.  Thanks :+1:

**Comment by jpechane** [comment]
@Naros Applied, thanks

**Pull request merged**